source/guides/guides/stubs-spies-and-clocks.md

```diff
@@ -39,9 +39,9 @@ You can refer to each of these libraries documentation for more examples and exp
 
 ## Stubs
 
-A stub is a way to modify a function and force its behavior to be controlled by you (the programmer).
+A stub is a way to modify a function and delegate control overs its behavior to you (the programmer).
```

Comment:
"force its behavior to be controlled by you" is unambiguous, but hard for my brain to process quickly.